<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThuPhiDialog</class>
 <widget class="QDialog" name="ThuPhiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thu Phí Đoàn Viên</string>
  </property>
  <widget class="QLabel" name="lblDoanVien">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Đoàn viên:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSoTien">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Số tiền:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSoTien">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNgayNop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày nộp:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateNgayNop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGhiChu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ghi chú:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGhiChu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLuu">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHuy">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDoanVien">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
